doc/modules/feature_selection.rst

```diff
@@ -210,7 +210,7 @@ settings, using the Lasso, while :class:`RandomizedLogisticRegression` uses the
 logistic regression and is suitable for classification tasks.  To get a full
 path of stability scores you can use :func:`lasso_stability_path`.
 
-.. figure:: ../auto_examples/linear_model/plot_sparse_recovery_003.png
+.. figure:: ../auto_examples/linear_model/images/plot_sparse_recovery_003.png
```

Comment:
BTW to make sure we get it right for a little while, the [stable](http://scikit-learn.org/stable/modules/feature_selection.html#randomized-sparse-models) show this:

![plot_sparse_recovery_0021](https://cloud.githubusercontent.com/assets/1680079/6486280/216103a6-c28a-11e4-97b5-9523700ff195.png)

whereas locally I get this one instead:

![plot_sparse_recovery_0031](https://cloud.githubusercontent.com/assets/1680079/6486305/49ba4a88-c28a-11e4-9334-3fa16a08230b.png)

Not sure which one is the more meaningful given the context.